src/PostgREST/Middleware.hs

```diff
@@ -41,6 +41,7 @@ runWithClaims conf eClaims app req =
         claimsSql = map (pgFmtEnvVar "request.jwt.claim.") [(c,unquoted v) | (c,v) <- M.toList claimsWithRole]
         setRoleSql = maybeToList $
           (\r -> "set local role " <> r <> ";") . toS . pgFmtLit . unquoted <$> M.lookup "role" claimsWithRole
+        setSchemaSql = ["set schema " <> pgFmtLit (configSchema conf) <> ";"] :: [Text]
```

Comment:
The postgres [docs](
https://www.postgresql.org/docs/10/static/sql-set.html) say

> SET SCHEMA 'value' is an alias for SET search_path TO value. Only one schema can be specified using this syntax.

To allow the full generality of comma-separated search path items, we could change `configSchema` to a list that is split at config parsing time, then do something like:

```haskell
setSchemaSql = [
    "SET search_path TO " <>
    (intercalate ',' . map pgFmtIdent $ configSchema conf)) <> ";"
  ] :: [Text]
```
I didn't test this snippet, but you get the idea.

Also after parsing the config we can check whether `configSchema` has more than one item, and issue that warning text discussed in a previous comment.

Reply:
@steve-chavez does this look good to you? If you're busy, I can write the code for detecting multiple schema items and issuing an error.

Reply:
The snippet is good but I think just adding the  `SET search_path TO` to the `Middleware.hs` won't work(tests will fail), there's some code to be removed, mainly the `Schema` prepending. I already had an idea of how to do this but didn't have the time, I think I could finish this PR on the coming week, if I cannot finish I'll pass the ball, is that ok?

Reply:
Thanks, and yeah of course.